docs/metrics.asciidoc

```diff
@@ -160,3 +160,56 @@ All metrics collected from `prometheus_client` are prefixed with `"prometheus.me
 [[prometheus-metricset-beta]]
 ===== Beta limitations
  * The metrics format may change without backwards compatibility in future releases.
+
+[float]
+[[custom-metrics]]
+=== Custom Metrics
```

Comment:
There's a jump table on L11-L13. We should add en entry that links to this new topic.